### PR TITLE
punx validate needs option to choose file set, default to latest available

### LINKED ISSUE #192: main: validate needs option to choose file set
The `punx validate` subcommand needs an option to choose which file set to use for validation.  The help needs to show which file set will be the default.

### PR DESCRIPTION
**Pull request by prjemian**
- FIX #192

### CODE REVIEW

**Comment by prjemian** [comment]
@carterbox Ready for review

File: docs/source/cmd_validate.rst [hunk 1]
```diff
@@ -32,18 +32,19 @@ validate a NeXus file
...
+..  code-block:: console
+    :linenos:
 
-   usage: punx validate [-h] [--report REPORT] infile
+    usage: punx validate [-h] [--report REPORT] infile
 
-   positional arguments:
-     infile           HDF5 or NXDL file name
+    positional arguments:
+      infile           HDF5 or NXDL file name
```
Comment (carterbox) [inline docs/source/cmd_validate.rst:41]: Was this added whitespace intentional?

File: docs/source/cmd_validate.rst [hunk 2]
```diff
@@ -32,18 +32,19 @@ validate a NeXus file
 
 .. rubric:: command line help
 
-.. code-block:: console
-   :linenos:
+..  code-block:: console
+    :linenos:
 
-   usage: punx validate [-h] [--report REPORT] infile
+    usage: punx validate [-h] [--report REPORT] infile
```
Comment (carterbox) [inline docs/source/cmd_validate.rst:38]:
```suggestion
    usage: punx validate [-h] [--report REPORT] [-f FILE_SET_NAME] infile
```

**Review by carterbox — APPROVED** [review]
I tried the new option on an example file. It seems to work as I would expect it to.

File: docs/source/cmd_validate.rst (hunk 1 above)
Comment (prjemian) [reply docs/source/cmd_validate.rst:41]: Intentional.  VSCode indents things on 4-columns and this one was on 3.  A few other places will see this as well.

**Comment by prjemian** [comment]
@carterbox : Thanks!

**Pull request merged**